### PR TITLE
Update default ofed driver

### PR DESCRIPTION
**Pull request by sharsonia**

### CODE REVIEW

File: XML/Other/ReplaceableTestParameters.xml [hunk 1]
```diff
@@ -670,7 +670,7 @@ Scenario 2 : You need to run all the performance tests quickly.
 	</Parameter>
 	<Parameter>
 		<ReplaceThis>MLX_OFED_PARTIAL_LINK</ReplaceThis>
-		<ReplaceWith>https://partnerpipelineshare.blob.core.windows.net/ofed/MLNX_OFED_LINUX-4.7-3.2.9.0-</ReplaceWith>
+		<ReplaceWith>https://partnerpipelineshare.blob.core.windows.net/ofed/MLNX_OFED_LINUX-5.4-1.0.3.0-</ReplaceWith>
```
Comment (LiliDeng) [inline XML/Other/ReplaceableTestParameters.xml]: have you uploaded this version into partnerpipelineshare?

Comment (sharsonia) [reply XML/Other/ReplaceableTestParameters.xml]: Yes uploaded now.

image

Comment (LiliDeng) [reply XML/Other/ReplaceableTestParameters.xml]: MLNX_OFED_LINUX-5.4-3.0.3.0-?

Comment (sharsonia) [reply XML/Other/ReplaceableTestParameters.xml]: yes. Long had suggested this version when I asked him about the pkey error

**Review by pupacha — APPROVED** [review]

**Pull request merged**